### PR TITLE
05-images

### PR DESCRIPTION
**Pull request by Gani4ka**

### CODE REVIEW

File: helpers/users-router-controller.js [hunk 1]
```diff
@@ -96,6 +96,33 @@ class usersRouterController {
       status: 200,
     };
   }
+
+  async patchUser(id, payload) {
+    this.result = await updateUser(id, payload);
+
+    if (this.result instanceof Error && this.result.errors)
```
Comment (odykyi) [inline helpers/users-router-controller.js]: use brackets {}

File: helpers/users-router-controller.js [hunk 2]
```diff
@@ -96,6 +96,33 @@ class usersRouterController {
...
+    this.result = await updateUser(id, payload);
+
+    if (this.result instanceof Error && this.result.errors)
+      return {
+        data: JSON.stringify({
+          message: this.result.errors || this.result.message,
+        }),
+        status: 400,
+      };
+    else if (this.result instanceof Error && this.result.message)
```
Comment (odykyi) [inline helpers/users-router-controller.js]: use brackets {}

File: users.js [hunk 3]
```diff
@@ -7,9 +12,24 @@ async function addUser(email, password) {
 
     if (isUserExist.length === 0) {
       const hash = bcrypt.hashSync(password, 10);
+
+      const size = 200;
+      const value = email;
+
+      const png = jdenticon.toPng(value, size);
+
+      const avatar = path.join(
```
Comment (odykyi) [inline users.js]: move path to config

example: 

export const yourFunction = () => `text-${id}`

File: helpers/multer.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const multer = require("multer");
+const path = require("path");
+const uuid = require("uuid").v4;
+
+const storage = multer.diskStorage({
+  destination: path.join(process.cwd(), "tmp"),
```
Comment (odykyi) [inline helpers/multer.js]: move path to config

**Review by odykyi — CHANGES_REQUESTED** [review]
comments

File: helpers/users-router-controller.js (hunk 1 above)
Comment (Gani4ka) [reply helpers/users-router-controller.js]: done

File: helpers/users-router-controller.js (hunk 2 above)
Comment (Gani4ka) [reply helpers/users-router-controller.js]: done

File: users.js (hunk 3 above)
Comment (Gani4ka) [reply users.js]: done

File: helpers/multer.js (hunk 4 above)
Comment (Gani4ka) [reply helpers/multer.js]: done

**Review by odykyi — APPROVED** [review]

**Pull request merged**